returnn/torch/data/pipeline.py

```diff
@@ -309,6 +311,99 @@ def __iter__(self):
             yield current_batch
 
 
+# noinspection PyAbstractClass
```

Comment:
Why is this needed? Is this an abstract class or not? I assume it is not, but then why do you get this warning?

Reply:
I get the warning that this class does not implement all abstract methods, probably because we're not implementing `reset()` (which we don't need to, since we're overriding `__iter__`). The same suppression is applied to the `BatchingIterDataPipe`.

Reply:
What PyTorch version? `reset()` is not an abstract method for me. It seems it's also not needed for `BatchingIterDataPipe`?

Reply:
The one in CI that is used when the PyCharm inspections run.

Reply:
Ok... I don't like this. But let's clean this up in a followup. First check that we maybe can increase the PyTorch version in CI.